anvio/drivers/hmmer.py

```diff
@@ -70,40 +99,24 @@ def run_hmmscan(self, source, alphabet, context, kind, domain, num_genes_in_mode
         self.run.info('Context', context)
         self.run.info('Domain', domain if domain else 'N\\A')
         self.run.info('HMM model path', hmm)
-        self.run.info('Number of genes', num_genes_in_model)
+        self.run.info('Number of genes in HMM model', num_genes_in_model)
         self.run.info('Noise cutoff term(s)', noise_cutoff_terms)
         self.run.info('Number of CPUs will be used for search', self.num_threads_to_use)
+        if alphabet in ['DNA', 'RNA']:
+            self.run.info('HMMer program used for search', 'nhmmscan')
+        else:
+            self.run.info('HMMer program used for search', self.program_to_use)
 
-        # we want to create hmm files in the same direcotry
         tmp_dir = os.path.dirname(self.target_files_dict[target][0])
-        log_file_path = os.path.join(tmp_dir, '00_log.txt')
+        log_file_path = os.path.join(tmp_dir, '*_log')
```

Comment:
I changed this because with multithreading, each hmmscan worker thread gets its own log. (and, even with just 1 thread there is never a log file called '00_log.txt'. Here is an example:
```
$ ls /var/folders/1n/2s6d_kq53pv9js812zwcljq80000gn/T/tmphz7r15hp/*log
/var/folders/1n/2s6d_kq53pv9js812zwcljq80000gn/T/tmphz7r15hp/AA_gene_sequences.fa.0_log /var/folders/1n/2s6d_kq53pv9js812zwcljq80000gn/T/tmphz7r15hp/AA_gene_sequences.fa.2_log
/var/folders/1n/2s6d_kq53pv9js812zwcljq80000gn/T/tmphz7r15hp/AA_gene_sequences.fa.1_log /var/folders/1n/2s6d_kq53pv9js812zwcljq80000gn/T/tmphz7r15hp/AA_gene_sequences.fa.3_log
```

This particular variable is used just for printing run info, whereas the actual log file names are generated later in the code (in the for loop that spawns the worker threads). So at the time it was easiest to just update `log_file_path`.

I changed it to have a literal `*` so that copying the file path into an `ls` command would allow you to list all the log files, but I do agree that it would be best to print the actual log file names for each thread. I will change that now.